courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
<p>if we're defining PROJECT above do we need to do it here too?</p><p> </p><p>Also, should there be TODOs or is the task just to replace Project and run cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>This is a python variable, above is a bash variable. Different scopes.</p>

Reply:
<p>I'll add a #TODO. By now, hopefully students will know the drill haha.</p>

Reply:
right. thanks.